### PR TITLE
Create static routes

### PR DESCRIPTION
**Pull request by cindyli**
* [X] This pull request has been linted by running `npm run lint` without errors
* [X] This pull request has been tested by running `npm run dev` and reviewing affected routes
* [X] This pull request has been built by running `npm run generate` without errors
* [X] This isn't a duplicate of an existing pull request

## Description

To resolve https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/90.

This pull request generate static pages for: the home page, /inclusion-challenges, /about, /news, /news/page/{page-num}, /views, /views/page/{pageNum}, /views/{a-post}, /tag/{tag-name}

## Steps to test

1. Run `npm run generate`;
2. In the output, the "Generating pages" section should generate all routes for pages mentioned above;
3. Run `npx serve dist` to test these pages.

**Expected behavior:**

Static pages should be generated.

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -25,10 +25,14 @@ export default {
...
 	},
 	data () {
-		const currentPage = this.$store.state.sitePages.filter(oneSitePage => oneSitePage.slug === "home")[0];
-		return {
+		const currentPage = this.$store.state.sitePages.find(oneSitePage => oneSitePage.slug === "home");
+
+		return currentPage ? {
 			title: currentPage.title,
 			content: currentPage.content
+		} : {
```
Comment (TeddTech) [inline pages/index.vue:33]: This stops the website from rendering the `layouts/error.vue` page

File: pages/index.vue [hunk 2]
```diff
@@ -25,10 +25,14 @@ export default {
...
+			title: "Not Found",
```
Comment (TeddTech) [inline pages/index.vue:34]: This stops the website from rendering the `layouts/error.vue` page

File: pages/_slug/index.vue [hunk 3]
```diff
@@ -11,10 +11,14 @@ export default {
...
 	data () {
-		const currentPage = this.$store.state.sitePages.filter(oneSitePage => oneSitePage.slug === this.$route.params.slug)[0];
-		return {
+		const currentPage = this.$store.state.sitePages.find(oneSitePage => oneSitePage.slug === this.$route.params.slug);
+
+		return currentPage ? {
 			title: currentPage.title,
 			content: currentPage.content
+		} : {
+			title: "Not Found",
```
Comment (TeddTech) [inline pages/_slug/index.vue:20]: This stops the website from rendering the `layouts/error.vue` page

Comment (cindyli) [reply pages/_slug/index.vue]: Good catch, @TeddTech. This is fixed in the new commit. Thanks.

**Comment by greatislander** [comment]
Looks great, @cindyli!

**Pull request merged**